middleware/decrypt.py

```diff
@@ -131,8 +133,8 @@ def get_args():
 def main():
     """Coordinate execution of script."""
     args = get_args()
-    logger.debug("file_paths: %s", ", ".join([f.name for f in args.file_paths]))
-    logger.debug("output_dir: %s", args.output_dir)
+    logger.debug(f"file paths: {", ".join([f.name for f in args.file_paths])}")
+    logger.debug(f"output directory: {args.output_dir}")
```

Comment:
Sorry to be nagging, but for consistency with your other log messages, I think it's better to start each message with a capital letter ;-)